docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,691 @@
+# ﻿Tizen 6.0 Public M1 Release Notes
+
+Release Date: May. 31, 2020

```

Comment:
@surbhig12 
For other months, it is correct to use a period as the month name is abbreviated. (August -> Aug. October -> Oct.)

In case of "May", it is not an abbreviation. So we don't need to use a period after "May".
I think the link you checked should be updated also, as it has a period even it was released in May. I'll do that later.

Reply:
will accept